modules/vardictjava/main.nf

```diff
@@ -0,0 +1,50 @@
+process VARDICTJAVA {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::vardict-java=1.8.3" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/vardict-java:1.8.3--hdfd78af_0':
+        'quay.io/biocontainers/vardict-java:1.8.3--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(bam), path(bai)
+    path(bed)
+    tuple path(fasta), path(fasta_fai)
+
+    output:
+    tuple val(meta), path("*.vcf.gz"), emit: vcf
+    path "versions.yml"              , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def args2 = task.ext.args2 ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    def VERSION = '1.8.3'
+
+    """
+    vardict-java \\
+        $args \\
+        -c 1 -S 2 -E 3 \\
+        -b $bam \\
+        -th $task.cpus \\
+        -N $prefix \\
+        -G $fasta \\
+        $bed \\
+        | teststrandbias.R \\
+        | var2vcf_valid.pl \\
+            $args2 \\
+            -N $prefix \\
```

Comment:
Is pipeline these between each other how the tool is meant to be used? As in, is `vardict-java` useless if you don't do so? Or can you save the output of testrandbias to a file that can then be sent to a separate `var2vcf_valid` module?

Reply:
It's not really useful to save the intermediate files, because vardict-java creates an internal tabular file, which gets converted to VCF by `var2vcf_valid.pl`

Reply:
Ok, if it's only useful internally then this is OK :+1: